idl/github.com/uber/cadence/shared.thrift

```diff
@@ -417,6 +423,7 @@ struct WorkflowExecutionContinuedAsNewEventAttributes {
   50: optional i32 executionStartToCloseTimeoutSeconds
   60: optional i32 taskStartToCloseTimeoutSeconds
   70: optional i64 (js.type = "Long") decisionTaskCompletedEventId
+  80: optional i32 backoffStartIntervalInSeconds
```

Comment:
How is it different from retryPolicy.initialIntervalSeconds?

Reply:
This is the backoff interval to create the first schedule event for next iteration (which may not be the first time of retry). The initialInterval is the backoff interval for first retry.